TEAMS.md

```diff
@@ -44,7 +44,7 @@
 
 ### Project Contributors
 
-[@AswinTimalsina][@AswinTimalsina], [@Brahyt][@Brahyt], [@mritunjaysharma394][@mritunjaysharma394]
+[@AswinTimalsina][@AswinTimalsina], [@Brahyt][@Brahyt], [@mritunjaysharma394][@mritunjaysharma394], [@joshwlewis][@joshwlewis]
```

Comment:
nit: we should probably keep them alphabetical.

Reply:
```suggestion
[@AswinTimalsina][@AswinTimalsina], [@Brahyt][@Brahyt], [@joshwlewis][@joshwlewis], [@mritunjaysharma394][@mritunjaysharma394]
```